src/components/vr-scatter-plot/vr-scatter-plot.component.ts

```diff
@@ -44,13 +49,15 @@ export class VRScatterPlotComponent<T extends Meta>implements OnInit, OnChanges,
     .subscribe(dataset => {
       // componentMetas is initialized to different dataset metas - will help funnel dataset
       this.componentMetas = dataset.metas;
-      // dataset.metas[0].type = 'tabbed' and dataset.metas[1] = 'line'
+      // dataset.metas[0].type = 'tabbed' and dataset.metas[1] = 'line' if chosen UserWhiteNoise
+      // dataset.metas[0] - 'line' if Dummy chosen
       this.meta = dataset.metas[0];
       if (this.isTabbed()) {
+       // this.meta = dataset.metas[0];
         this.meta2 = (this.meta as any).metas[0];
-        console.log(this.meta2);
       } else {
-        this.meta2 = this.meta[0];
+        this.meta = dataset.metas;
+        this.meta2 = this.meta[0] as LineChartMeta;
```

Comment:
It seems like `meta2` simply references a part of `meta`.

In that case, instead of creating a separate variable for `meta2, can you just make a quicker accessor method that extracts the correct content from `meta` and use that instead? Generally, want to avoid having overlapping variables.

Reply:
https://github.com/googleinterns/guide-doge/pull/57#discussion_r456192492